soh/soh/Enhancements/audio/AudioEditor.cpp

```diff
@@ -306,19 +305,11 @@ void DrawTypeChip(SeqType type) {
     ImGui::EndDisabled();
 }
 
-void DrawSfxEditor(bool& open) {
-    if (!open) {
-        if (CVarGetInteger("gAudioEditor.WindowOpen", 0)) {
-            CVarClear("gAudioEditor.WindowOpen");
-            LUS::RequestCvarSaveOnNextTick();
-        }
-        return;
-    }
-
+void AudioEditor::DrawElement() {
     AudioCollection::Instance->InitializeShufflePool();
 
     ImGui::SetNextWindowSize(ImVec2(820, 630), ImGuiCond_FirstUseEver);
-    if (!ImGui::Begin("Audio Editor", &open)) {
+    if (!ImGui::Begin("Audio Editor", &mIsVisible)) {
```

Comment:
Does `&mIsVisible` come from the super? I don't see it defined anywhere

Reply:
yes

https://github.com/Kenix3/libultraship/blob/6d29ff2dbc4507b59771d5af84cb58eaacd2f9a7/src/window/gui/GuiElement.h#L30

but we probably should clean this up to not do early returning within `DrawElement` because that should be handled here now

https://github.com/Kenix3/libultraship/blob/6d29ff2dbc4507b59771d5af84cb58eaacd2f9a7/src/window/gui/GuiElement.cpp#L35-L43